yarn.lock

```diff
@@ -4937,7 +4937,7 @@
     "@typescript-eslint/types" "4.30.0"
     eslint-visitor-keys "^2.0.0"
 
-"@uniswap/default-token-list@^2.0.0", "@uniswap/default-token-list@^2.1.0":
+"@uniswap/default-token-list@^2.0.0":
```

Comment:
mistake?

Reply:
intentional: the deleted version is from an explicit dependency in the package.json which was not actual used, and so deleted. (this was found with `npx yarn-deduplicate`, but manually verified.)